### ISSUE TITLE
Problem when setting high width.

### ISSUE THREAD

**Issue by DiegoDevBittencourt** [teodosii/react-notifications-component#67]
Sup guys! So this is my code:

```
import { store } from 'react-notifications-component';

import 'react-notifications-component/dist/theme.css'
import 'animate.css'

export default Notification = {

  dangerNotification: function (message) {

    store.addNotification({
      message: message,
      type: "danger",
      insert: "top",
      container: "top-right",
      animationOut: ["animated", "fadeOut"],
      width: 600,
      dismiss: {
        duration: 4000,
        onScreen: true,
        showIcon: true,
      },
    });
  }
}
```

And this is how the notification appears on my screen:

![not2](https://user-images.githubusercontent.com/21102506/80920339-9036ed00-8d45-11ea-8415-100807689e22.png)

I want that the notification appears like this:

![not1](https://user-images.githubusercontent.com/21102506/80920349-a6dd4400-8d45-11ea-828b-510ede08d10f.png)

But bigger, the problem is that when I change the width size to 350+ it will cut the right edge. So, how can I fix this? Also the ```<ReactNotification />``` are inside my ```<app/>``` component in root.

**Comment by teodosii** [comment]
Hey, could you please post a gif to illustrate this issue? The container should be properly placed and the notification should take width accordingly to the container.
@DiegoBittencourtOfficial

**Comment by DiegoABittencourt** [comment]
@teodosii here it go:
![ezgif-2-6786c93cb540](https://user-images.githubusercontent.com/57305119/90860464-6c350100-e360-11ea-98ac-1bb5f677dd18.gif)

**Comment by teodosii** [comment]
Containers have changed in `3.0.0` and are quite dynamic in terms of width. Fixed in `3.0.0`
